### PR TITLE
enhancement/natitoches_so_cprr_2018_2021

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

**Comment by ayyubibrahimi** [comment]
@pckhoi - Another confusing process-data fail as `/data/fuse/per_natchitoches_so.csv` exists and runs fine on local.

File: fuse/natchitoches_so.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+if __name__ == "__main__":
+    cprr = pd.read_csv(deba.data("match/cprr_natchitoches_so_2018_2021.csv"))
+    agency = cprr.agency[0]
+    post = load_for_agency(agency)
+    per_df = fuse_personnel(cprr, post)
+    com_df = rearrange_allegation_columns(cprr)
+    event_df = fuse_events(cprr, post)
+    event_df.to_csv(deba.data("fuse/event_natchitoches_so.csv"), index=False)
+    com_df.to_csv(deba.data("fuse/com_natchitoches_so.csv"), index=False)
+    per_df.to_csv(deba.data("fuse/per_natchitoches_so.csv"), index=False)
```
Comment (pckhoi) [inline fuse/natchitoches_so.py:59]: This script `fuse/natchitoches_so.py` does not appear to run. Please check that the file `.deba/deps/fuse.d` list this script.

Comment (ayyubibrahimi) [reply fuse/natchitoches_so.py]: It wasn't there before. Confused as to why. It is currently there. 

```
$(DEBA_DATA_DIR)/fuse/event_natchitoches_so.csv $(DEBA_DATA_DIR)/fuse/com_natchitoches_so.csv $(DEBA_DATA_DIR)/fuse/per_natchitoches_so.csv &: $(DEBA_MD5_DIR)/fuse/natchitoches_so.py.md5 $(DEBA_DATA_DIR)/match/cprr_natchitoches_so_2018_2021.csv $(DEBA_DATA_DIR)/clean/pprr_post_2020_11_06.csv $(DEBA_MD5_DIR)/datavalid.yml.md5 | $(DEBA_DATA_DIR)/fuse
	$(call deba_execute,fuse/natchitoches_so.py)
```

**Comment by ayyubibrahimi** [comment]
@rajivsinclair Here is an example of a `process-data` error

**Comment by pckhoi** [comment]
When I try to run make on local:
```bash
make: *** No rule to make target 'data/ocr/advocate_post_ohr_pdfs.csv', needed by 'data/ner/advocate_post_officer_history_reports.csv'.  Stop.
```
Please fix this first before worrying about GH action.

**Comment by ayyubibrahimi** [comment]
Apologies. This bug was addressed in PR #342. Please attempt a re-run.

**Comment by ayyubibrahimi** [comment]
Please attempt to re-run this branch.

**Comment by pckhoi** [comment]
There was a bug with Deba v0.1.7 where it can only pick up the first dependency in a function call. Which causes problem in a script such as `fuse/all.py`:
```python
...
def fuse_personnel():
    return rearrange_personnel_columns(
        pd.concat(
            [
                pd.read_csv(deba.data("fuse/per_baton_rouge_pd.csv")),
                pd.read_csv(deba.data("fuse/per_baton_rouge_so.csv")),
                pd.read_csv(deba.data("fuse/per_new_orleans_harbor_pd.csv")),
                pd.read_csv(deba.data("fuse/per_new_orleans_pd.csv")),
                ...
```
Upgrade to Deba v0.1.8 will fix this.

**Comment by pckhoi** [comment]
@ayyubibrahimi This is not a Github action problem. Again, this is a problem you can reproduce on local. How can you tell? The error message in the process-data job is a Make error. Please learn more about Make, learn to detect its errors, learn to read `Makefile`, `wrgl.mk`, `deba.mk`, and all files under `.deba/deps`. If Deba (or any of my libraries) misbehaves, then I will fix of course.

**Comment by pckhoi** [comment]
For now just update deba to 0.1.8 in `requirements.txt` for all branches.

**Comment by ayyubibrahimi** [comment]
Thanks for the find. I'll do a better job of being able to re-produce these errors on local, and understanding Make. 

I updated deba to 0.1.8 in `requirements.txt` in PR #343 and merged it with `main` to include update deba for all branches.

`main` has just been merged with this branch, however, the process-data is still occurring.

**Comment by pckhoi** [comment]
Please follow the instructions within `gh_k8s_runner/README.md`. You have to update image whenever requirements are updated.

**Comment by ayyubibrahimi** [comment]
Thanks. I ran `gh_k8s_runner/install.sh excellent-zoo-300106` and received the following output where it looks as if nothing has changed, despite the changes made to `requirements.txt`. I will look into the `WARNING` below.

```
...
b40ed86654e5: Layer already exists
6277e28519c2: digest: sha256:df992dc17b7404497984b6d226d468f24f981a299a24d82310cc3bbc1fc1b997 size: 4304
W0913 04:46:54.707850   23668 gcp.go:119] WARNING: the gcp auth plugin is deprecated in v1.22+, unavailable in v1.26+; use gcloud instead.
To learn more, consult https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke
namespace/cdi-runner unchanged
persistentvolumeclaim/cdi-runner-data unchanged
runnerdeployment.actions.summerwind.dev/cdi-runner configured
```

**Comment by pckhoi** [comment]
```
runnerdeployment.actions.summerwind.dev/cdi-runner configured
```
It did update alright. You need to manually re-run job after the deployment is updated.

**Comment by pckhoi** [comment]
Actually looks like there's another issue. Will push a fix soon.

**Comment by pckhoi** [comment]
@ayyubibrahimi the pipeline run fine now. Please check transaction to see if that's what you expected.

**Pull request merged**